node-operators/block-producers/offboarding/.pages

```diff
@@ -0,0 +1,5 @@
+title: Offboarding
+hide: false
+nav:
+  - index.md
+  - 'Decommission Block Producer': 'account.md'
```

Comment:
```suggestion
  - 'Decommission Your Account': 'account.md'
```

I wanted to put "Decommission Your Block Producer," so it sort of aligns with the rest of the titles, but thats too long -- it gets broken into two lines. So I'd prefer "Decommission Your Account" for the left nav